charts/newrelic-infrastructure/README.md

```diff
@@ -1,29 +1,119 @@
+![Version: 3.3.2](https://img.shields.io/badge/Version-3.3.2-informational?style=flat-square) ![AppVersion: 3.1.1](https://img.shields.io/badge/AppVersion-3.1.1-informational?style=flat-square)
 
 A Helm chart to deploy the New Relic Kubernetes monitoring solution
 
 **Homepage:** <https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration/>
 
+# Helm installation
+
+You can install this chart using [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) located in the
+[helm-charts repository](https://github.com/newrelic/helm-charts) or directly from this repository by adding this Helm repository:
+
+```shell
+helm repo add nri-kube-events https://newrelic.github.io/nri-kube-events
+helm upgrade --install nri-kube-events/nri-kube-events -f your-custom-values.yaml
+```
+
 ## Source Code
 
 * <https://github.com/newrelic/nri-kubernetes/>
-* <https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure>
+* <https://github.com/newrelic/nri-kubernetes/tree/master/charts/newrelic-infrastructure>
+* <https://github.com/newrelic/infrastructure-agent/>
+
+## Values managed globally
+
+This chart implements the [New Relic's common Helm library](https://github.com/newrelic/helm-charts/tree/master/library/common-library) which
+means that it honors a wide range of defaults and globals common to most New Relic Helm charts.
+
+Options that can be defined globally include `affinity`, `nodeSelector`, `tolerations`, `proxy` and others. The full list can be found at
+[user's guide of the common library](https://github.com/newrelic/helm-charts/blob/master/library/common-library/README.md).
+
+## Chart particularities
+
+### Low data mode
+There are two mechanisms to reduce the amount of data that this integration sends to New Relic. See this snippet from the `values.yaml` file:
+```yaml
+common:
+  config:
+    interval: 15s
+
+lowDataMode: false
+```
+
+The `lowDataMode` toggle is the simplest way to reduce data send to Newrelic. Setting it to `true` changes the default scrape interval from 15 seconds
+(the default) to 30 seconds.
+
+If you need for some reason to fine-tune the number of seconds you can use `common.config.interval` directly. If you take a look at the `values.yaml`
+file, the value there is `nil`. If any value is set there, the `lowDataMode` toggle is ignored as this value takes precedence.
+
+Setting this interval above 40 seconds can make you experience issues with the Kubernetes Cluster Explorer so this chart limits setting the interval
+inside the range of 10 to 40 seconds.
+
+### Affinities and tolerations
+
+The New Relic common library allows to set affinities, tolerations, and node selectors globally with the rest of the integrations and products but this
+chart has an extra level of granularity to set them.
```

Comment:
```suggestion
When leveraged in a chart, the New Relic common library allows to set affinities, tolerations, and node selectors globally. However, this
chart has an extra level of granularity that allows to configure them for each ones of the components.
```

I am not sure about this, but the old sentence was quite confusing

Reply:
Solved in [#3d77275](https://github.com/newrelic/nri-kubernetes/pull/409/commits/3d77275993abcbfdd3bff2660bc3b64e18abf656)

Reply:
Solved in [#85e36f5](https://github.com/newrelic/nri-kubernetes/pull/409/commits/85e36f5115754dce005f8970e931c32630797de5)